lib/tasks/demo.rake

```diff
@@ -329,10 +328,17 @@ namespace :demo do
       end
 
       # Potential Exposure Info
-      patient[:continuous_exposure] = rand < 0.3
-      patient[:last_date_of_exposure] = today - rand(5).days unless patient[:continuous_exposure]
+      patient[:isolation] = days_ago > 10 ? rand < 0.9 : rand < 0.4
+      if patient[:isolation]
+        patient[:symptom_onset] = today - rand(10).days
+        patient[:user_defined_symptom_onset] = true
+      else
+        patient[:continuous_exposure] = rand < 0.3
+        patient[:last_date_of_exposure] = today - rand(5).days unless patient[:continuous_exposure]
+      end
       patient[:potential_exposure_location] = Faker::Address.city if rand < 0.7
       patient[:potential_exposure_country] = Faker::Address.country if rand < 0.8
+      patient[:exposure_notes] = Faker::Games::LeagueOfLegends.quote if rand < 0.5
```

Comment:
Hahaha I looked at some examples of this in my console and found exactly the kind of thing I was looking for:
```
"Not Draven; Draaaaven."
```

Reply:
This is a really fun library to work with https://github.com/faker-ruby/faker#tv-shows https://github.com/faker-ruby/faker#games